sections/home/Slider/Slider.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useState } from "react";
+import Image from "next/image";
+import styles from "./Slider.module.css";
+import CustomButton from "@/components/ui/CustomButton/CustomButton";
+import moreIcon from "../../../public/icons/Vector.png";
+import CustomSwiper from "../../../components/ui/CustomSwiper/CustomSwiper";
+import ArrowButton from "../../../components/ui/CustomSwiper/ArrowButton";
+
+const Sliderpage: React.FC = () => {
```

Comment:
use camel case for components name, also avoid using the word Page in components